indico/web/client/js/ckeditor.js

```diff
@@ -0,0 +1,108 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2022 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+export const getConfig = ({simple = true, images = false} = {}) => ({
+  language: 'en_GB',
+  fontFamily: {
+    options: [
+      'Sans Serif/"Liberation Sans", sans-serif',
+      'Serif/"Liberation Serif", serif',
+      'Monospace/"Liberation Mono", monospace',
+    ],
+  },
+  toolbar: {
+    shouldNotGroupWhenFull: false,
+    items: [
+      'heading',
+      '|',
+      'bold',
+      'italic',
+      'strikethrough',
+      '|',
+      'fontColor',
+      'removeFormat',
+      '|',
+      'bulletedList',
+      'numberedList',
+      'outdent',
+      'indent',
+      '|',
+      'link',
+      !simple && images && 'imageInsert',
+      !simple && 'insertTable',
```

Comment:
When adding a table I get a raw `<figure class="table"></figure>` in the displayed minutes. Probably this isn't whitelisted in our bleach config. Let's make sure that anything the editor inserts is on the HTML whitelist (for this one it seems perfectly safe to allow) to avoid ugly HTML showing up